CONTRIBUTING.md

```diff
@@ -1,25 +1,58 @@
 # Contributing to Hazelcast
 
-It makes you feel good...
+Hazelcast is Open Source software, licensed under the [Apache 2.0. license](LICENSE).
+The main benefit of Open Source is that you don't need to wait for a vendor to provide a fix or a feature.
+If you've got the skills (and the will), it's already at your fingertips.
+
+There are multiple ways to contribute:
+
+1. [Reporting an issue](#issue-reports)
+2. [Sending a pull request](#pull-requests).
+Note that you don't need to be a developer to help us.
+Contributions that improve the documentation are always appreciated.
 
```

Comment:
I would mention also reaching us via Slack to discuss if needed. 

```suggestion

If you need any assistance or brainstorming, create a GH issue to discuss or reach us directly via [Slack](https://slack.hazelcast.com/).
```

Reply:
I'm much more in favor of Slack instead of GH. IMHO, there are already too many people creating issues that shouldn't be created in the first place.